cinema/views.py

```diff
@@ -2,9 +2,16 @@
 
 from django.db.models import F, Count
 from rest_framework import viewsets
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.mixins import ListModelMixin, \
```

Comment:
When you have a long tape of imports, it is better to put all these imports in brackets and make a transfer